java/ql/lib/semmle/code/java/Expr.qll

```diff
@@ -1831,37 +1844,43 @@ class ExtensionReceiverAccess extends VarAccess {
 }
 
 /**
- * An l-value is a write access to a variable, which occurs as the destination of an assignment.
+ * A write access to a variable, which occurs as the destination of an assignment.
  */
-class LValue extends VarAccess {
-  LValue() { this.isLValue() }
+class VarWrite extends VarAccess {
+  VarWrite() { this.isVarWrite() }
 
   /**
-   * Gets a source expression used in an assignment to this l-value.
+   * Gets the right-hand side of the assignment that executes this variable write.
```

Comment:
This change to the top-level one-liner is incorrect, as is also explained a few lines below, i.e. compound assignments include the LHS as a source. It's quite unfortunate that the predicate name is `getRhs`, as that's similarly wrong.